### PR TITLE
[Feat]:variety of Cards component added

### PR DESCRIPTION
**Pull request by t007rushi**
This is a PR for card components.
added cards with the following variations

simple card
Cards with badge
Cards with dismiss
Cards with text overlay
Text only cards
Horizontal card
card with shadow

### CODE REVIEW

**Review by horsemaker — COMMENTED** [review]
lookout for your class names.

**Comment by nikhiljugale007** [comment]
Card is not distinguishable from the background. You can add border or elevation or some box-shadow

**Review by ajit-sharmaGH — APPROVED** [review]
Everything Looks cool 😎 !

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  font-size: 0.6rem;
+  margin-left: 0.3rem;
+}
+.saving {
+  margin-left: 0.4rem;
+  color: var(--card-txt-color);
+  font-size: 0.8rem;
+}
+
+.opa {
```
Comment (JantuDeb) [inline components/card.css]: You can  use something like this for this class names `.opa-1/2` 
In CSS you need to escape the slash
`.opa-1\/2{
opacity:0.4;
}
`

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  opacity: 0.8;
+}
+
+.bg-color p {
+  margin-top: 0;
+  font-size: 1.2rem;
+  color: var(--card-txt-colr);
+}
+
+.top {
```
Comment (JantuDeb) [inline components/card.css:163]: You can use `top-0` for this class

**Review by JantuDeb — APPROVED** [review]
I added some suggestions you can get a look at it.

**Review by horsemaker — APPROVED** [review]

File: components/card.css (hunk 1 above)
Comment (t007rushi) [reply components/card.css]: changed to transparent though

**Pull request merged**